src/pages/softwareSkills.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "../styles/softwareSkills.module.scss";
+import {skillsSection} from "../portfolio";
+
+export default function SoftwareSkill() {
+  return (
+    <div>
+      <div className="software-skills-main-div">
+        <ul className="dev-icons">
+          {skillsSection.softwareSkills.map((skills, i) => {
+            return (
+              <li
+                key={i}
+                className="software-skill-inline"
+                name={skills.skillName}
+              >
+                <i className={skills.fontAwesomeClassname}></i>
+                <p>{skills.skillName}</p>
+              </li>
+            );
+          })}
+        </ul>
+      </div>
+    </div>
+  );
+}
+
+// export async function getStaticProps() {
+//   return {
```

Comment:
No need to create a page inside the pages components you need to create seprate component for this

Reply:
got it working on it right now